org.mixedrealitytoolkit.input/Interactors/Ray/MRTKRayInteractor.cs

```diff
@@ -46,7 +89,27 @@ public class MRTKRayInteractor :
         /// Used to check if the parent controller is tracked or not
         /// Hopefully this becomes part of the base Unity XRI API.
         /// </summary>
-        private bool IsTracked => xrController.currentControllerState.inputTrackingState.HasPositionAndRotation();
+        private bool IsTracked
+        {
+            get
+            {
+                #pragma warning disable CS0618 // Type or member is obsolete
```

Comment:
Change the order of operations here, so `#pragma warning disable CS0618` only surrounds `xrController` usage and not `TrackedPoseDriver`.  Limit the scope of disabling the warning

Reply:
done in https://github.com/MixedRealityToolkit/MixedRealityToolkit-Unity/pull/750/commits/17305f6a751cc7715f888db852aaebace17f6dd9 , resolving